From d9d0c1ec9fd77cddb613394de033cb0b7b8ff925 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 16 Apr 2010 15:58:43 -0400
Subject: [PATCH 031/635] revert vsync wait due the change in epdc cimmit
 change Signed-off-by: guoyin.chen
 <guoyin.chen@freescale.com>

---
 libgralloc/framebuffer.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libgralloc/framebuffer.cpp b/libgralloc/framebuffer.cpp
index 157b44e..5dd12e0 100755
--- a/libgralloc/framebuffer.cpp
+++ b/libgralloc/framebuffer.cpp
@@ -164,6 +164,8 @@ static int fb_post(struct framebuffer_device_t* dev, buffer_handle_t buffer)
             m->base.unlock(&m->base, buffer); 
             return -errno;
         }
+        unsigned int ret;
+        ioctl(m->framebuffer->fd, MXCFB_WAIT_FOR_VSYNC, &ret);
 
         #ifdef FSL_EPDC_FB
         if(ctx->partial_update) {
-- 
1.8.0

